src/VersionManager.cs

```diff
@@ -19,68 +20,166 @@ internal class VersionManager
         /// <summary>
         /// The latest version of DarkRift.
         /// </summary>
-        private static string latestDarkRiftVersion;
+        private static Version latestDarkRiftVersion;
 
         /// <summary>
         /// Gets the path to a specified installation, downloading it if required.
         /// </summary>
         /// <param name="version">The version number required.</param>
         /// <param name="pro">Whether the pro version should be used.</param>
-        /// <param name="netStandard">Whether the .NET Standard build should be used.</param>
+        /// <param name="platform">Whether the .NET Standard build should be used.</param>
         /// <returns>The path to the installation, or null, if it cannot be provided.</returns>
-        public static string GetInstallationPath(string version, ServerTier tier, ServerPlatform platform)
+        public static string GetInstallationPath(Version version, ServerTier tier, ServerPlatform platform)
```

Comment:
The documentation on this method states that it `downloads it if required` still. That's not true anymore.